src/rpc/rpc_schema.js

```diff
@@ -25,14 +26,18 @@ const VALID_HTTP_METHODS = {
 class RpcSchema {
 
     constructor() {
-        this._ajv = new Ajv({ verbose: true, schemaId: 'auto', allErrors: true });
-        this._ajv.addKeyword('idate', schema_utils.KEYWORDS.idate);
-        this._ajv.addKeyword('objectid', schema_utils.KEYWORDS.objectid);
+        this._ajv = new Ajv({ verbose: true, allErrors: true });
```

Comment:
Does `strict: true` work now that you have keywords for `methods` and `doc` ?

Reply:
No, once adding the Keywords that are missing (methods and doc) we don't need `strict: false` anymore.
but the default is `log` and not `true`
From Ajv code:
```
export interface CurrentOptions {
    strict?: boolean | "log";
```
So I have tested using `strict: true` and it fails.